### PR TITLE
You cannot use the short version of the functions in the serverless-s…

### PR DESCRIPTION
**Pull request by jforest**
…tep-functions plugin, it's not supported at this time

### CODE REVIEW

File: serverless-stepf.yml [hunk 1]
```diff
@@ -27,7 +29,8 @@ rBuilds:
             Next: Finished
       Finished:
         Type: Task
-        Resource: arn:aws:lambda:#{AWS::Region}:#{AWS::AccountId}:function:${self:service}-${self:provider.stage}-finished
+        Resource:
+          Fn::Sub: arn:aws:lambda:${AWS::Region}:${AWS::AccountId}:function:${self:service}-${self:provider.stage}-finished
```
Comment (jspiewak) [inline serverless-stepf.yml:33]: Is this going to work okay, with some things being substituted by serverless first, and some being substituted by CloudFormation?

Comment (jforest) [reply serverless-stepf.yml:33]: I don't know, I have that same question, haven't found an answer yet

Comment (jforest) [reply serverless-stepf.yml:33]: When I merge, it will deploy to staging which is fine, so I can find out quickly without breaking production

Comment (jspiewak) [reply serverless-stepf.yml:33]: 🤞🏻

Comment (jforest) [reply serverless-stepf.yml:33]: image
🎉 the answer was yes!

**Pull request merged**